packages/fxa-auth-server/scripts/populate-vat-taxes.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import { ACTIVE_SUBSCRIPTION_STATUSES } from 'fxa-shared/subscriptions/stripe';
+import Stripe from 'stripe';
+
+import { setupProcesingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+const pckg = require('../package.json');
+
+export async function init() {
+  // Load program options
+  program.version(pckg.version).parse(process.argv);
+
+  const { log, stripeHelper } = await setupProcesingTaskObjects();
+
+  const stripe = (stripeHelper as any).stripe as Stripe;
+
+  // Locate all plans using a VAT currency
+  const plans = (await stripe.plans.list({ limit: 100 })).data.filter((plan) =>
+    ['eur', 'chf'].includes(plan.currency)
+  );
+
+  for (const activeStatus of ACTIVE_SUBSCRIPTION_STATUSES) {
+    for (const plan of plans) {
+      for await (const sub of stripe.subscriptions.list({
+        limit: 100,
```

Comment:
Why only 100 subscriptions per plan?

Reply:
This is an async iterator, 100 is the max per 'page' of results per async iteration.

Reply:
Not sure what it is about this interface that keep tripping me up.  I made a similar comment in a previous SubPlat script PR.  🙄